### PR TITLE
return functionality for books in adminview

### LINKED ISSUE #360: Add return book functionality
### 📇 User story
As a user I want to be able to return a book I have loaned.

### 📜 Acceptance Criteria
A list of changes that need to be implemented to consider this feature done.
- [ ] Book is removed from students active loans.

### PR DESCRIPTION
**Pull request by TE4-Rasmus-Brednert**
## ⚙️ What issue did you implement or fix?
Fixes #360 

## 📜 Summary
- Fixed functionality for returning books in adminview.

## 📝 Type of change

Please delete options that are not relevant.

- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Change bar into zar. :shipit:

### CODE REVIEW

**Review by te4-alexander-kjellberg — APPROVED** [review]
Seems legit.

File: frontend/src/views/AdminLoansView.vue [hunk 1]
```diff
@@ -4,31 +4,41 @@
...
-      </div>
-      <div>
-        <LoanListComponent />
+  <div>  
+    <div>
+      <h1 class="display-2">
+        Loans
+      </h1>
+      <div class="flex">
+        <div style="margin-right: 50px">
```
Comment (te4-alexander-nylundgomes) [inline frontend/src/views/AdminLoansView.vue:13]: Perhaps this should follow vue structure and be in the bottom in a style tag?

File: frontend/src/views/AdminLoansView.vue [hunk 2]
```diff
@@ -4,31 +4,41 @@
...
+          <LoanFormComponent />
+        </div>
+        <div>
+          <LoanListComponent />
+        </div>
       </div>
     </div>
+    <div style="margin-top: 50px">
```
Comment (te4-alexander-nylundgomes) [inline frontend/src/views/AdminLoansView.vue:21]: Same as above

File: frontend/src/views/AdminLoansView.vue [hunk 3]
```diff
@@ -4,31 +4,41 @@
...
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import LoanFormComponent from '@/components/LoanFormComponent.vue';
 import LoanListComponent from '@/components/LoanListComponent.vue';
+import ReturnLoanComponent from '@/components/ReturnLoanComponent.vue';
 import LoansModule from '../store/modules/LoansModule';
 
 @Component({
   components: {
     LoanFormComponent,
-    LoanListComponent
```
Comment (te4-alexander-nylundgomes) [inline frontend/src/views/AdminLoansView.vue:31]: Is LoanListComponent still in use? If not, perhaps the import of the component should be removed?

**Review by te4-alexander-nylundgomes — CHANGES_REQUESTED** [review]
Looks good but could use some structure

File: frontend/src/views/AdminLoansView.vue (hunk 3 above)
Comment (TE4-Rasmus-Brednert) [reply frontend/src/views/AdminLoansView.vue:31]: yes, it is in use.

**Review by te4-alexander-nylundgomes — APPROVED** [review]
Great!

**Pull request merged**